tests/end_to_end/install.sh

```diff
@@ -25,5 +25,23 @@ install_bats() {
   cd -
 }
 
+install_paifs() {
+  cp -r ../../../pai-fs ./
+  cd pai-fs
+  pip install -r requirements.txt
+  cd -
+}
+
+prepare_cntk_job() {
+  git clone https://github.com/Microsoft/CNTK.git
+}
+
+
 apt-get install -y dos2unix
-install_bats
\ No newline at end of file
+
+mkdir -p local
+cd local
+install_bats
+install_paifs
+prepare_cntk_job
+cd -
```

Comment:
How about we put the testing code and dependencies into a Docker image and execute there in the next step?

Reply:
make it as an option?